CMakeLists.txt

```diff
@@ -200,6 +200,7 @@ FIND_PACKAGE(GDAL)
 FIND_PACKAGE(Expat REQUIRED)
 FIND_PACKAGE(Spatialindex REQUIRED)
 FIND_PACKAGE(Qwt REQUIRED)
+FIND_PACKAGE(QJSON)
```

Comment:
Sure - I just noted that there was a way to avoid the extra dependency. And IIUC QJSON is going to be replaced with Qt5Core functionality anyway.